java/arcs/core/crdt/extension/ConversionExtensions.kt

```diff
@@ -29,11 +28,7 @@ private fun Any?.toReferencable(): Referencable {
   return when {
     ReferencablePrimitive.isSupportedPrimitive(this::class) -> {
       if (this is ByteArray) {
-        ReferencablePrimitive(
-          ByteArray::class,
-          this,
-          valueRepr = Base64.encode(this)
-        )
+        ReferencablePrimitive(ByteArray::class, this)
```

Comment:
this method is unused (it's private and not being used here) so you can just get rid of it.